src/irmin-pack/inode.ml

```diff
@@ -851,6 +918,11 @@ struct
       stabilize Total t
 
     let save layout ~add ~mem t =
+      let clear =
```

Comment:
we don't want to expose this as a flag?

Reply:
Ideally it would be exposed and be forwarded from `Tree.export ?clear` through `P.Node.add`, but it seems like a big change for not much.

I've improved the comment at that point of the code.